requirements.txt

```diff
@@ -1,5 +1,5 @@
 appdirs~=1.4
-cached-property~=1.4.3
+cached-property~=1.0.0
```

Comment:
I think this change is incorrect. It should be ~=1.0.

Reply:
@ironbatshashank Have you tested for versions lower than this? Does the tests still pass when you lower the version than `1.0.0`. That's what the issues asks about. The issues ask you to find out if any older version of `cached-property` works. Please lower the dependency and check if the build still passes.

Reply:
@Naveenaidu is correct. Also, the change that you made excludes `1.3.0` and `1.4.3`, which definitely works. jayvdb has already checked for both.

Reply:
Thanks for the guidance @shashank-b @Naveenaidu @bnu123 . I'll submit another PR with these implementations.